modules/scheduler/gke-cluster/main.tf

```diff
@@ -267,6 +267,17 @@ resource "google_container_node_pool" "system_node_pools" {
   }
 }
 
+### TODO: remove this after Terraform support for GKE Parallelstore CSI is added. ###
+###       Instead use addons_config above to enable the CSI                       ###
+resource "null_resource" "enable_parallelstore_csi" {
+  count = var.enable_parallelstore_csi == true ? 1 : 0
+
+  provisioner "local-exec" {
```

Comment:
Is the cluster already ready to accept these changes via gcloud ? Also does it require Toolkit customer to install gcloud as a dependency ?

Reply:
Can we add that in README, also can we add that in some sort of validation/requirements file ?

Reply:
this is a temporary workaround, we will be replacing this with addons_config in the google_container_cluster after tf support is added(tracked in b/341371408). I am more incline to leave it as minimal as possible for easier clean up. WDYT?

Reply:
So perhaps asking this a different way, if the customer doesn't have gcloud installed, what kind of error do they get ? 
Also would they know which version of gcloud is sufficient, I am trying to avoid surprises to customers.

Reply:
they would get, gcloud command not found. 
TBH I will be very surprise customer have installed tf, toolkit, but not gcloud. And in a lot of our output instruction we are including the step of gcloud run as well.